sandpack-react/src/components/CodeEditor/index.tsx

```diff
@@ -25,11 +26,12 @@ export const SandpackCodeEditor: React.FC<CodeEditorProps> = ({
   showLineNumbers = false,
   showRunButton = true,
   wrapContent = false,
+  closableTabs = false,
 }) => {
   const { sandpack } = useSandpack();
   const { code, updateCode } = useActiveCode();
   const { activePath, status, editorState } = sandpack;
-  const shouldShowTabs = showTabs ?? sandpack.openPaths.length > 1;
+  const shouldShowTabs = showTabs ?? sandpack.openPaths.length > 0;
```

Comment:
Is there any reason why you have changed this? I think it doesn't belong to the closable tabs.

Reply:
Yup, reverting. In the first design i made `clossable` as default. It was needed at the time. Good catch 😄